boefjes/boefjes/worker/repository.py

```diff
@@ -169,7 +173,7 @@ def _cached_resolve_boefjes(path: Path) -> dict[str, BoefjeResource]:
     for path, package in paths_and_packages:
         try:
             boefje_resources.append(get_boefje_resource(path, package, hash_path(path)))
-        except ModuleException as exc:
+        except (ModuleException, ValidationError) as exc:
             logger.exception(exc)
```

Comment:
Not changed in this PR, but I just spotted it. `logger.exception` already adds the exception to the log message, so we shouldn't also use the exception as log message.

```suggestion
            logger.exception("Error getting boefje resource")
```